docker/base.dockerfile

```diff
@@ -2,7 +2,7 @@
 FROM faasm/cpython:0.1.0 as python
 
 # Import from SGX container
-FROM faasm/sgx:0.5.3 as sgx
+FROM faasm/sgx:0.8.1 as sgx
```

Comment:
How come this isn't `0.8.2`? Looks like the `faasm/sgx` image has changed in this build.

Reply:
The SGX image needs to be re-built manually with `inv docker.build -c sgx` and it picks up whatever version your working environment is in.

I bumped the code version last, what means I generated the image with the `0.8.1` version. I can re-generate it again with the new tag, but the code version makes no difference in the image build.

Reply:
Ah yes ok, because the image doesn't actually check out the code it doesn't really matter. Makes sense.